src/Containers/packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -121,7 +122,8 @@
 
     <PropertyGroup>
       <ContainerGenerateLabels Condition="'$(ContainerGenerateLabels)' == ''">true</ContainerGenerateLabels>
-      <ContainerGenerateLabelsImageCreated Condition="'$(ContainerGenerateLabelsImageCreated)' == ''">true</ContainerGenerateLabelsImageCreated>
+      <!-- NOTE: this label screws up incrementality - may want to disable it by default -->
+      <ContainerGenerateLabelsImageCreated Condition="'$(ContainerGenerateLabelsImageCreated)' != 'true'">false</ContainerGenerateLabelsImageCreated>
```

Comment:
Could you use the last write timestamp of the main output assembly instead?  That should be close enough to the actual date and is better than nothing.

I've found the created label very useful when troubleshooting people running `latest` tag, not realising it may not be the same `latest` as the container registry, so it would be a shame to loose it by default.

Reply:
That's a really interesting idea - System.IO.File.GetLastWriteTime is a static method so we can easily call it from the MSBuild logic to power this!